### PR TITLE
chore: remove Execa

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by Nirajn2311 — APPROVED** [review]

**Review by Nirajn2311 — CHANGES_REQUESTED** [review]
My bad, pnpm hadn't installed the packages properly. The `precypress` command fails when using the updated package because the import fails due to the package being pure ESM.

**Comment by Nirajn2311** [comment]
Yea that sounds like a good idea. We can do what execa does with node itself easily.

File: cypress-install.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
+const { spawn } = require('child_process');
 const pkg = require('cypress/package.json');
-const execa = require('execa');
 
-const version = pkg.version;
+console.log('Installing Cypress version:', pkg.version);
```
Comment (ojeytonwilliams) [inline cypress-install.js:5]:
```suggestion
```
Cypress itself logs this, so we don't have to.

File: cypress-install.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
+const { spawn } = require('child_process');
 const pkg = require('cypress/package.json');
```
Comment (ojeytonwilliams) [inline cypress-install.js:2]:
```suggestion
```
Unnecessary if we're not logging.

File: cypress-install.js [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
-    env: { CYPRESS_INSTALL_BINARY: version }
-  });
-  console.log('Cypress installed');
-})();
+const child = spawn('pnpm', ['run', 'cypress:install'], {
+  stdio: 'inherit'
+});
+
+child.on('close', code => {
+  console.log('Cypress installation finished with code:', code);
```
Comment (ojeytonwilliams) [inline cypress-install.js:11]:
```suggestion
  if (code) {
    console.error('Cypress installation failed with code:', code);
  }
```
Again, Cypress logs for us, so we don't need to log if it worked.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
There's a lot of unrelated changes in the lockfile, could you revert the changes and just `pnpm remove execa`?

**Comment by Sembauke** [comment]
This should do it @ojeytonwilliams

**Comment by Sembauke** [comment]
Never mind I will open a fresh pr for this one.

**Comment by ojeytonwilliams** [comment]
There's no need to open a new PR - it just creates noise.  The only reason to close this would be if we're not intended to make the change (and we are).

**Pull request closed without merging**